motion_computation/src/motion_computation_worker.cpp

```diff
@@ -189,8 +190,23 @@ void MotionComputationWorker::psmCallback(const carma_v2x_msgs::msg::PSM::Unique
     return;
   }
 
-  // TODO Include same conversion and id checking logic as in mobility path
-  // psm_list_.objects.push_back(mobilityPathToExternalObject(msg));
+  // TODO temporary info
+  std::string map_frame_id = "map";
```

Comment:
This can probably be made a member variable of motion_computation

Reply:
Should this be a config param?

Reply:
Sure, though it won't change at any point in the near future.